.claude/hooks/utils/llm/anth.py

```
@@ -0,0 +1,114 @@
+#!/usr/bin/env -S uv run --script
```

Ruff (0.12.2)

Shebang is present but file is not executable

(EXE001)